### DISCUSSION TITLE
Why does the halfmove clock matter here for evaluation

### DISCUSSION THREAD

**Discussion by dlbbld** [official-stockfish/Stockfish#3907] [Q&A] [answered]
For the below FEN, the only difference is the halfmove clock and the fullmove counter.

But the best move for Black is a capture, so I would assume the difference in the halfmove clock does not matter. The difference in the fullmove counter does not matter of course. But the evaluations are different, like -28 and -6 at for latest Stockfish at depth 26.

Can somebody please explain to me why the difference in halfmove clock results in a different evaluation?

<pre>
r2qnrk1/3nbppp/3pb3/4pPP1/pp1NP3/4B3/PPPQN2P/2KR1B1R b - - 1 16
r2qnrk1/3nbppp/3pb3/4pPP1/pp1NP3/4B3/PPPQN2P/2KR1B1R b - - 10 21
</pre>

**Comment by vondele** [comment, marked as answer]
the halfmove clock is used in the evaluation of positions statically, in particular scales the evaluation a bit down as one goes towards the 50 moves rule counter. The static evaluation doesn't know the moves available, and definitely not the bestmoves. After that, any difference in evaluation will lead to a very different search tree, and eventually small differences in the search evaluation of the position.

**Reply by dlbbld** [reply]
Thanks a lot for the explanation.
